### DISCUSSION TITLE
"HomeConnectError: insufficient_scope: Insufficient scope for this resource"

### DISCUSSION THREAD

**Discussion by nicedevil007** [alexkn/node-red-contrib-homeconnect#70] [Q&A]
I get the error message from the title.

I'm using it with a node like this:
![image](https://user-images.githubusercontent.com/17103076/180844849-8fa93be6-d72e-4779-8a55-4709fef06e4b.png)

What am I doing wrong?

```
{
  "data": {
    "key": "BSH.Common.Setting.PowerState",
    "value": "BSH.Common.EnumType.PowerState.On",
    "type": "BSH.Common.EnumType.PowerState",
    "constraints": {
      "allowedvalues": [
        "BSH.Common.EnumType.PowerState.On",
        "BSH.Common.EnumType.PowerState.Standby"
      ]
    }
  }
}
```

**Comment by alexkn** [comment]
check the authorized scopes

**Comment by nicedevil007** [comment]
What do you mean?
I already did the authorization process with an "OK" at the end.

![image](https://user-images.githubusercontent.com/17103076/180847398-41a95579-a3b7-4957-9c38-676bcf8c78b4.png)

EDIT: I guess you mean this? But nothing more I can do here or?

![image](https://user-images.githubusercontent.com/17103076/180848088-aeea8289-551d-4678-953a-6396dab08b60.png)

**Comment by nicedevil007** [comment]
I did a complete removal of my nodered app on the home-connect page and recreated it with new clientID and secretID.
Didn't do the trick :(

**Comment by nicedevil007** [comment]
Next thing I tryed out even with all scopes enabled:
opened: https://apiclient.home-connect.com/

Clicked on 
![image](https://user-images.githubusercontent.com/17103076/180857386-72a984ce-f481-4e7f-b2b3-5c47b6a65f90.png) entered my clientID and secretID here and selected all and hit Authorize in this window again.

Result:

![image](https://user-images.githubusercontent.com/17103076/180857645-4ee0c750-b6ee-40af-8051-afc898c7d06d.png)

**Reply by gusman80** [reply]
did u get this done? i had the some problem and found out i had to add both scopes: "IdentifyAppliance Dishwasher"
